templates/bot-msgext/ts/.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Any file in this folder will also be added to user's project. Do we really need to put eslint config here?

Reply:
The files in the "default" folder will be added. This one won't.

Reply:
Yes. You are right.